tests/middleware/test_logging.py

```diff
@@ -45,6 +46,54 @@ async def test_trace_logging(caplog):
         assert "ASGI [2] Completed" in messages.pop(0)
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize("protocol", [("h11"), ("httptools")])
+async def test_trace_logging_on_http_protocol(protocol, caplog):
+    config = Config(app=app, log_level="trace", http=protocol)
+    with caplog_for_logger(caplog, "uvicorn.protocol"):
```

Comment:
```suggestion
    with caplog_for_logger(caplog, "uvicorn.error"):
```
I merged your PR about that, should be good with the "correct" logger name ?

Reply:
Yes. Since GH-1086 has been merged, I could set the correct logger name.